index.html

```diff
@@ -55,7 +55,7 @@ <h2>第２回Git勉強会</h2>
                     <article>
                         <h3>イベント日時・場所</h3>
                         <p>3月23日 19:00開始</p>
-                        <p>株式会社〇〇 イベントセミナー会場</p>
+                        <p>株式会社インプレス イベントセミナー会場</p>
```

Comment:
```suggestion
                        <p>株式会社インプレス！！！ イベントセミナー会場</p>
```
この方がインプレスさんにとってうれしくないですか？

Reply:
それはない笑

Reply:
変えてください！！